lib/csetgrp.gi

```diff
@@ -749,6 +762,137 @@ local l;
+          #p:=PositionProperty(orbset,x->e in x);
+          p:=LookupDictionary(dict,e);
+          if p<>fail and
+            # we could have found an element that we know (because of
+            # fusion) to be already in this orbit (but must store)
+            pam[map[p]]<>this then
+            p:=map[p]; #retrieved image position might have been fused away
+            return p;
+          fi;
+          Add(o,img);
+          AddSet(os,e);
+          if p=fail then AddDictionary(dict,e,this);fi;
+        fi;
+      od;
+      i:=i+1;
+    od;
+    #if i>Length(o) and Length(os)>Length(o) then Error("ran out of orbit");fi;
+    return -(i-1);
+  end;
+
+  bahn:=[];
+  enum:=Enumerator(P);
+  live:=[];
+  orbs:=[];
+  orbset:=[];
+  done:=[];
+  dict:=NewDictionary(1,true,rec(hashfun:=x->x));
+  map:=[];
+  pam:=[]; # reverse of map
+  for nr in [1..Length(reps)] do
+    #p:=PositionProperty(orbset,x->Position(enum,reps[nr]) in x);
+    p:=LookupDictionary(dict,Position(enum,reps[nr]));
+    if p=fail then
+      # start orbit algorithm
+      o:=[reps[nr]];
+      os:=[Position(enum,reps[nr])];
+      AddDictionary(dict,os[1],nr);
+      p:=orbitextender(o,os,1,lim,nr);
+      if p<0 then 
+        # new orbit
+        Info(InfoCoset,4,nr," lives");
+        Add(live,nr);
+        Add(orbs,o);
+        Add(orbset,os);
+        Add(done,-p);
+        Add(bahn,[nr]);
+        map[nr]:=Length(orbs);
+        pam[Length(orbs)]:=nr;
+        i:=1;
+        while Length(orbs)>count do
+          # one orbit too many
+          if ForAll(orbs,x->Length(x)>=lim) then 
+            if lim<20000 then
+              lim:=lim*2;
+            else
+              lim:=(QuoInt(lim,8000)+1)*8000;
+            fi;
+          fi;
+          Info(InfoCoset,4,"Redo ",i," ",lim);
+          p:=orbitextender(orbs[i],orbset[i],done[i],lim,pam[i]);
+          if p>0 then
+            Info(InfoCoset,4,"Join ",i," to ",p);
+            if p=i then Error("selfjoin cannot happen");fi;
+            bahn[p]:=Union(bahn[p],bahn[i]);
+
+            #UniteSet(orbset[p],orbset[i]);
+            for j in [1..Length(map)] do
+              if IsBound(map[j]) and map[j]=i then map[j]:=p; fi;
+            od;
+
+            # delete entry i, move higher ones one up
+            for j in [1..Length(map)] do
+              if IsBound(map[j]) and map[j]>i then map[j]:=map[j]-1; fi;
+            od;
+
+<<<<<<< HEAD
+            # Remove entry i, i.e.
+            #p:=[i..Length(orbs)-1];
+            #orbs{p}:=orbs{p+1};Unbind(orbs[Length(orbs)]);
+=======
+            #p:=[i..Length(orbs)-1];
+            #orbs{p}:=orbs{p+1};Unbind(orbs[Length(orbs)]);
+            #orbset{p}:=orbset{p+1};Unbind(orbset[Length(orbset)]);
+            #done{p}:=done{p+1};Unbind(done[Length(done)]);
+            #bahn{p}:=bahn{p+1};Unbind(bahn[Length(bahn)]);
+            #pam{p}:=pam{p+1};Unbind(pam[Length(pam)]);
+>>>>>>> 57ac0bd2e... Added `sisyphus` option to force extra work for testing
```

Comment:
You have commited merge conflict markers here, which is why this PR fails in all tests

Reply:
Thank you -- Something must have gone wrong in the last rebase. Is (I hope) fixed now.